features/step_definitions/assertions_steps.rb

```diff
@@ -1,3 +1,7 @@
 Then("I should see {string}") do |content|
     expect(page).to have_content content
+end
+
+Then("I should see the {string} page") do |page|
```

Comment:
you should not use page as a variable name, page object points to the page you are on and with this test you asking if `page = page`

Reply:
done